c7n/resources/artifact.py

```diff
@@ -88,6 +91,44 @@ def _remove_repositories(self, client, domain):
                 ignore_err_codes=('ResourceNotFoundException',))
 
 
+@ArtifactDomain.filter_registry.register('cel')
```

Comment:
Consolidate this class with ArtifactRepoCELFilter class -- see glue.py CEL filter for example